app/controllers/save_pages_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SavePagesController < ApplicationController
+  include AccountConcern
+
+  before_action :set_no_cache_headers
+  before_action :authenticate
+
+  def create
+    redirect_to GdsApi.account_api.save_page(
+      page_path: params[:page_path],
+      govuk_account_session: account_session_header,
+    ).to_h["saved_page"]["page_path"]
```

Comment:
I think we'll want to pass some indication of success back to the page, so we can display a success box.

We have a similar need in the Transition Checker (for showing the success box when you save your results), but there we do it with the slightly dodgy approach of checking if the timestamp of the latest results is only a short while ago.  Maybe we should standardise on a query parameter to use for passing personalisation-related flash notices?

So maybe this could be something like:

```ruby
GdsApi.account_api.save_page(page_path: params[:page_path], govuk_account_session: account_session_header)
redirect_to params[:page_path] + "?personalisation=page_saved"
```